src/components/Tooltip/index.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+const Tooltip = styled.div`
+    display: ${props => (props.active ? 'block' : 'none')};
+    position: absolute;
+    top: -52px;
+    left: -4px;
+    width: 108px;
```

Comment:
As I said before, these fixed values should be dynamic, I think that you could change this to `min-width`